TECHMANIA/Assets/Scripts/DiscordController.cs

```diff
@@ -16,9 +20,67 @@ public static void RunCallbacks ()
         discord.RunCallbacks();
     }
 
-    public static void SetActivity (Discord.Activity activity)
+    public static void SetActivity (string type)
     {
         if (discord == null || !SupportedOnCurrentPlatform()) return;
+        bool timestamp = false;
+        switch (type)
+        {
+            case "Main Menu":
```

Comment:
This code will be easier to maintain if these cases are enums (eg. `DiscordActivityType.MainMenu`) instead of hard-coded strings. Then, `lastActivityType` should be an enum as well.